spacy/kb.pyx

```diff
@@ -97,6 +93,7 @@ cdef class KnowledgeBase:
         self.vocab = vocab
         self.vocab.strings.add("")
         self._create_empty_vectors(dummy_hash=self.vocab.strings[""])
+        self._added_strings = [""]
```

Comment:
This should definitely be a set, not a list. We also don't need the empty string, that's built-in.
```suggestion
        self._added_strings = set()
```